### PR TITLE
Setup class for better flex calling MavenCommand

### PR DESCRIPTION
**Pull request by radtriste**
Include back https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/112 with fixes

### CODE REVIEW

**Comment by radtriste** [comment]
@Ginxo I created a small Jenkinsfile that load current content as dynamic library to be tested.
So we can test the MavenCommand can be correctly executed on Jenkins for example. What about configuring a PR check in Jenkins that run that Jenkinsfile ? :)

**Comment by radtriste** [comment]
successful Jenkinsfile run: https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/tradisso/job/test-setup-maven-command/23/console

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+local_lib=""
+
+pipeline {
```
Comment (rsynek) [inline Jenkinsfile:3]: Looks like the first integration test of the shared library :+1:

**Review by rsynek — APPROVED** [review]

**Comment by radtriste** [comment]
@Ginxo can you please have a look to this and merge if ok ?
That time it has been tested directly in Jenkins :)

**Review by Ginxo — APPROVED** [review]
congrats @radtriste !!! my only concern here is about Jenkinsfile location. Is root folder the right place to store it?
Cheers!

**Comment by radtriste** [comment]
Usually it is. but we could create a `.jenkins` folder if you prefer. After that, it is just a matter of configuration of the Jenkins job ;)

**Pull request merged**